app/controllers/users_controller.rb

```diff
@@ -152,10 +152,15 @@ def download_profile
   end
 
   def export_photos
-    tar_path = PhotoMover::move_photos(current_user)
-    send_data( File.open(tar_path).read, :filename => "#{current_user.id}.tar" )
+    current_user.queue_export_photos
+    flash[:notice] = I18n.t('users.edit.export_photos_in_progress')
+    redirect_to edit_user_path
   end
 
+  def download_photos
+    send_data File.open(current_user.exported_photos_file.path).read, filename: current_user.exported_photos_file.filename
```

Comment:
I thought about using send_file but works for local files, fails on remote.

If remote then it's better to redirect.

What do you think?